MIGRATION.md

```diff
@@ -21,6 +21,10 @@
   - [Framework-specific changes](#framework-specific-changes)
     - [React](#react)
       - [`react-docgen` component analysis by default](#react-docgen-component-analysis-by-default)
+    - [Removed postinstall](#removed-postinstall-1)
+    - [Removed stories.json](#removed-storiesjson-1)
+    - [StoryStore methods deprecated](#storystore-methods-deprecated)
+  - [Framework-specific changes](#framework-specific-changes-1)
```

Comment:
this looks like a badly resolved merge conflict

Reply:
Hmm, yeah, any advice? Should I rebase on `next`?

Reply:
I guess? My git-fu isn't great. In the rare times I've encountered something like this, I've just started a new branch and cherry picked everything over 🤷‍♂️